CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- There is always Unreleased section on the top. Subsections (Added, Changed, Fixed, Removed) should be added as needed. -->
 
 ## Unreleased
+- Updated PHPDoc
```

Comment:
we dont use past sentense in the change log - it should be in present

Reply:
Wording has been fixed